core/types/crosslink.go

```diff
@@ -4,51 +4,62 @@ import (
 	"math/big"
 	"sort"
 
-	"github.com/ethereum/go-ethereum/rlp"
-
 	"github.com/ethereum/go-ethereum/common"
-
+	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/harmony-one/harmony/block"
 )
 
 // CrossLink is only used on beacon chain to store the hash links from other shards
+// signature and bitmap correspond to |blockNumber|parentHash| byte array
+// Captial to enable rlp encoding
 type CrossLink struct {
-	ChainHeader *block.Header
+	ParentHashF  common.Hash
```

Comment:
What is "F" suffix for?

Reply:
Field, just used to distinguish between the method from the field.